revops/index.html

```diff
@@ -162,6 +165,36 @@ <h4>deranged is trusted by:</h4>
   </div>
 </section>
 
+<section class="who-we-are-section">
+  <div class="who-we-are-content">
+    <h2 class="who-we-are-section__title">Meet the team</h2>
```

Comment:
Do we use BEM-style class naming here? Then the section should be the Block `.who-we-are` followed by Elements `.who-we-are__content` and `.who-we-are__title`.

But these classes are not used in this commit, why are they there?

Reply:
Copied from frontpage I think. They definitely have styling if you look on the page in the preview :-)